src/components/Sidebar.js

```diff
@@ -130,12 +142,30 @@ export default ({
 }) => {
   const [location] = useLocation()
   const { user, isAuthed, logout } = useAuth()
+  const linksDict = ['General', 'Tools', 'Information', '']
   const links = [
-    { location: '/', text: 'DASHBOARD' },
-    { location: '/schedule', text: 'SCHEDULE' },
-    { location: '/quicklinks', text: 'QUICKLINKS' },
-    { location: '/faq', text: 'FAQ' },
-    { location: '/sponsors', text: 'SPONSORS' },
+    // General
+    [
+      { location: '/quicklinks', text: 'Getting Started' },
```

Comment:
What do you think of making `links` a dictionary where the keys are the categories (eg. "General", "Tools")? The values in this case would just be the array of link objects as you have them.

Imo would make the code below cleaner (replaces `links[1].push(...)` with `links.tools.push(...)` which is a _bit_ more descriptive, but this is more of a suggestion than anything!

Reply:
100%. I was boosted when I committed this code

Reply:
no you were not king 😤